### PR TITLE
Adding metrics.nodes to api model

### PR DESCRIPTION
**Pull request by zvikorn**
Signed-off-by: Tzvika Avni <tavni@redhat.com>

### CODE REVIEW

File: model/clusters_mgmt/v1/node_info_type.model [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Provides information about a node from specific type in the cluster.
+struct NodeInfo {
+	// The amount of the nodes from this type.
+	Amount *int
```
Comment (vkareh) [inline model/clusters_mgmt/v1/node_info_type.model:20]: This should be `Amount Integer`

Comment (zvikorn) [reply model/clusters_mgmt/v1/node_info_type.model]: oops...

Comment (zvikorn) [reply model/clusters_mgmt/v1/node_info_type.model]: Got you. Thanks!

**Comment by vkareh** [comment]
thanks! LGTM

**Pull request merged**